extensions/amazon-lambda/deployment/src/main/resources/lambda/manage.sh

```diff
@@ -43,10 +43,10 @@ function cmd_update() {
     --zip-file ${ZIP_FILE}
 }
 
-FUNCTION_NAME=${resourceName}Function
+FUNCTION_NAME=${lambdaName}
 HANDLER=io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler::handleRequest
-RUNTIME=java8
-ZIP_FILE=fileb://target/${artifactId}-${version}-runner.jar
+RUNTIME=java11
+ZIP_FILE=fileb://target/function.zip
 
```

Comment:
manage.sh still has target hardcoded, for both JVM and native file locations.

I cloned the branch from your repository and ran the maven archetype, and verified for gradle it is still showing as target (and not build).